mad-dog/src/index.js

```diff
@@ -129,32 +132,15 @@ async function main () {
       break
     }
     case 'test': {
-      const test = makeTest('consistency', coreIntegration, {
-        numCreatorNodes: DEFAULT_NUM_CREATOR_NODES,
-        numUsers: DEFAULT_NUM_USERS
-      })
-      await testRunner([test])
-      break
-    }
-    case 'test-snapback': {
-      const snapbackNumUsers = 40
-      const test = makeTest(
-        'snapback',
-        snapbackSMParallelSyncTest,
-        {
-          numUsers: snapbackNumUsers
-        }
-      )
-      await testRunner([test])
-      break
-    }
-    case 'test-ci': {
       const coreIntegrationTests = makeTest('consistency:ci', coreIntegration, {
         numCreatorNodes: DEFAULT_NUM_CREATOR_NODES,
         numUsers: DEFAULT_NUM_USERS
       })
 
-      // dynamically create ipld tests
+      const snapbackTest = makeTest('snapback', snapbackSMParallelSyncTest, {
```

Comment:
I spent some time thinking about this and while I think it may be valuable to consolidate into a single command - that is unrelated to the core of this task and kind of out of scope. Also having consulted with a few others, I think folks do use the existing `test` and `test-snapback` tests. Let's leave update the definition for `npm run test-ci` to include snapback per your changes here, but re-enable functionality for just running `npm run test` = coreIntegration test, and `npm run test-snapback` = snapbackTest